deployments/values/supported-matrix.yaml

```diff
@@ -251,10 +251,10 @@ matrix:
       version: "7.2"
       pattern: *ga_hotfixes_pattern
     insight-zeppelin:
-      version: "2.0.3"
+      version: *search_ga_version
```

Comment:
https://docs.alfresco.com/content-services/7.2/support/ I checked the cos it is set to 2.0.3 is it outdated?